src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,60 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    public static final String SUPPLY = "supply";
+    public static final String BUY = "buy";
+    public static final int SUPPLY_COUNT = 0;
+    public static final int BUY_COUNT = 1;
+    public static final int ACTION_INFO = 0;
+    public static final int ACTION_PRICE_INFO = 1;
+
     public void getStatistic(String fromFileName, String toFileName) {
+        writeReport(getReport(readFile(fromFileName)), toFileName);
```

Comment:
you divide only for write process, but you read and prepare a report inside, it look not good